README.md

```diff
@@ -14,12 +14,12 @@ In this task, we use the [Bulma CSS framework](https://bulma.io/). It is already
 1. `ClearButton` should clear the selection by setting an empty string to `selectedGood`.
 1. The title should show `No goods selected` when there is no selected good.
 1. Each good should have an `AddButton` to select the good.
-    - only 1 good can be selected at a time;
+   - only 1 good can be selected at a time;
 1. Don't show `AddButton` when a good is selected.
 1. Show `RemoveButton` for the selected good to clear the selection.
 
 ## Instructions
 
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_goods-selector-js/) and add it to the PR description.
+- Replace `oliaalia` with your Github username in the [DEMO LINK](https://oliaalia.github.io/react_goods-selector-js/) and add it to the PR description.
```

Comment:
Please add the demo link to the PR description
<img width="955" alt="image" src="https://github.com/mate-academy/react_goods-selector-js/assets/64750958/d0a23c0f-87a7-40f4-b609-736f32097a30">